kubernetes-ts-helm-release-wordpress/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "wordpress",
+  "devDependencies": {
+    "@types/node": "^12.0.0"
+  },
+  "dependencies": {
+    "@pulumi/kubernetes": "^3.0.0",
```

Comment:
Probably need to pin this to a recent dep for this to work.

Reply:
Bumped to 3.15.0